package.json

```diff
@@ -37,8 +36,17 @@
     "winston": "^3.2.1"
   },
   "devDependencies": {
+    "@grafana/eslint-config": "^2.5.0",
     "@types/express": "^4.11.1",
     "@types/node": "^14.14.41",
+    "@typescript-eslint/eslint-plugin": "^4.32.0",
+    "@typescript-eslint/parser": "^4.32.0",
+    "eslint": "^7.32.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-jsdoc": "^36.1.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "eslint-plugin-react": "^7.26.1",
+    "eslint-plugin-react-hooks": "^4.2.0",
```

Comment:
Not sure we need these since we're not using react here, the same goes for jsdoc plugin.

Reply:
I agree, we don't really need them, but they are required when using this `"extends": ["@grafana/eslint-config"]`.

I honestly wondered if it was worth it or if we should use our own config and preferred to use the Grafana config and keep these extra dependencies. WDYT?

Reply:
Ah I see, I guess we can keep these extra dependencies, they're dev dependencies anyways, so we don't need to maintain a separate config for the image renderer.